<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeMangeWidget</class>
 <widget class="QWidget" name="DeMangeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DeUserWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
    
	font: 16pt &quot;新宋体&quot;;
	background-color: rgb(244, 244, 244);
     border: 1px solid #8f8f91;
     border-radius: 6px;
     min-width: 100px;
     min-height: 30px;
 }

QTabWidget
{
   border:none;
}

 QPushButton:pressed {
	background-color: rgb(255, 255, 255);
 }

QTabWidget::tab-bar {
     alignment: center;
 }


 QTabBar::tab {
     background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #E1E1E1, stop: 0.4 #DDDDDD,
                                 stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);
     border: 1px solid #C4C4C3;
     border-bottom-color: #C2C7CB; /* same as the pane color */
     border-top-left-radius:  3px;
     border-top-right-radius: 3px;    
     padding-left: 20px;
     padding-right: 20px;
     padding-top: 10px;
     padding-bottom:6px;
 }

 QTabBar::tab:selected, QTabBar::tab:hover {
     background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #fafafa, stop: 0.4 #f4f4f4,
                                 stop: 0.5 #e7e7e7, stop: 1.0 #fafafa);
 }

 QTabBar::tab:selected {
     border-color: #9B9B9B;
     border-bottom-color: #C2C7CB; /* same as pane color */
 }

 QTabBar::tab:!selected {
     margin-top: 2px; /* make non-selected tabs look smaller */
 }</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="horizontalSpacing">
    <number>0</number>
   </property>
   <property name="verticalSpacing">
    <number>2</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget_3" native="true">
     <property name="styleSheet">
      <string notr="true">QPushButton
{   
   background-color: rgb(255, 255, 255, 0);
   border:0px solid red;
   border-radius:0px;
   font:bold 14px;
}

QPushButton:focus
{
   border-left-width:0px;
   border-right-width:0px;
   border-top-width:0px;
   border-bottom-width:2px;   
   border-bottom-color:  rgb(66, 189, 170);
   color:  rgb(66, 189, 170);
}</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="margin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QPushButton" name="funcBtn">
        <property name="text">
         <string>功能选择</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>108</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="userBtn">
        <property name="text">
         <string>用户管理</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="radioBtn">
        <property name="text">
         <string>放射源管理</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>108</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="5">
       <widget class="QPushButton" name="logoutBtn">
        <property name="text">
         <string>注销</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>用户管理</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="2">
          <widget class="QPushButton" name="delUserBtn">
           <property name="minimumSize">
            <size>
             <width>102</width>
             <height>32</height>
            </size>
           </property>
           <property name="text">
            <string>删除用户</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="changeUserBtn">
           <property name="minimumSize">
            <size>
             <width>102</width>
             <height>32</height>
            </size>
           </property>
           <property name="text">
            <string>修改用户</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="addUserBtn">
           <property name="minimumSize">
            <size>
             <width>102</width>
             <height>32</height>
            </size>
           </property>
           <property name="text">
            <string>添加用户</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="DeTableView" name="widget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>放射源管理</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="2">
          <widget class="QPushButton" name="delRadioactivityBtn">
           <property name="minimumSize">
            <size>
             <width>102</width>
             <height>32</height>
            </size>
           </property>
           <property name="text">
            <string>删除记录</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="modifyRadioactivityBtn">
           <property name="minimumSize">
            <size>
             <width>102</width>
             <height>32</height>
            </size>
           </property>
           <property name="text">
            <string>修改记录</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="addRadioactivityBtn">
           <property name="minimumSize">
            <size>
             <width>102</width>
             <height>32</height>
            </size>
           </property>
           <property name="text">
            <string>添加记录</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="DeTableView" name="widget_2" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DeTableView</class>
   <extends>QWidget</extends>
   <header location="global">detableview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
